test/compress/template-string.js

```diff
@@ -0,0 +1,36 @@
+template_strings: {
+    input: {
+        ``;
+        `xx\`x`;
+        `${ foo + 2 }`;
+        ` foo ${ bar + `baz ${ qux }` }`;
+    }
+    expect_exact: "``;`xx\\`x`;`${foo+2}`;` foo ${bar+`baz ${qux}`}`;";
+}
+
+template_string_prefixes: {
+    input: {
+        String.raw`foo`;
+        foo `bar`;
+    }
+    expect_exact: "String.raw`foo`;foo`bar`;";
+}
+
+template_strings_ascii_only: {
+    beautify = {ascii_only: true}
+    input: {
+        var foo = `foo
+        bar
+        ↂωↂ`
+    }
+    expect_exact: "var foo=`foo\\n        bar\\n        \u2182\u03c9\u2182`;"
```

Comment:
Unicode characters not escaped correctly - `\u` should be `\\u` in the context of being embedded in a double quoted expect_exact string. 

See: https://github.com/mishoo/UglifyJS2/blob/9676167aac715d2dd10918e2beb13b1a4a372616/test/compress/ascii.js#L15